diff --git a/node_modules/electron-dl/index.js b/node_modules/electron-dl/index.js
index c74d6b0..e96f286 100644
--- a/node_modules/electron-dl/index.js
+++ b/node_modules/electron-dl/index.js
@@ -55,6 +55,10 @@ function registerListener(session, options, callback = () => {}) {
 			item.setSavePath(filePath);
 		}
 
+		if (options.saveAs) {
+			item.setSaveDialogOptions({ defaultPath: filePath });
+		}
+
 		if (typeof options.onStarted === 'function') {
 			options.onStarted(item);
 		}
@@ -109,15 +113,15 @@ function registerListener(session, options, callback = () => {}) {
 					options.onCancel(item);
 				}
 			} else if (state === 'interrupted') {
-				const message = pupa(errorMessage, {filename: path.basename(filePath)});
+				const message = pupa(errorMessage, {filename: path.basename(item.getSavePath())});
 				callback(new Error(message));
 			} else if (state === 'completed') {
 				if (process.platform === 'darwin') {
-					app.dock.downloadFinished(filePath);
+					app.dock.downloadFinished(item.getSavePath());
 				}
 
 				if (options.openFolderWhenDone) {
-					shell.showItemInFolder(filePath);
+					shell.showItemInFolder(item.getSavePath());
 				}
 
 				callback(null, item);
